pkg/expansion/system_test.go

```diff
@@ -83,10 +83,21 @@ func TestExpand(t *testing.T) {
 		},
 		{
 			name:      "expand with nil namespace returns error",
-			generator: fixtures.LoadFixture(fixtures.DeploymentNginx, t),
+			generator: fixtures.LoadFixture(fixtures.DeploymentNginxWithNs, t),
 			ns:        nil,
 			mutators: []types.Mutator{
-				fixtures.LoadAssign(fixtures.AssignPullImage, t),
+				fixtures.LoadAssign(fixtures.AssignPullImageWithNs, t),
+			},
+			templates: []*expansionunversioned.ExpansionTemplate{
+				fixtures.LoadTemplate(fixtures.TempExpDeploymentExpandsPods, t),
+			},
+			expectErr: true,
+		},
+		{
+			name:      "expand with nil namespace returns error bc the namespace selector errs out",
```

Comment:
we need to update these test names since they are not `nil namespaces`?

Reply:
ah great catch! this test `name` should be flipped with the one above. Let me correct them.